backend/src/users/users.controller.ts

```diff
@@ -12,6 +12,17 @@ export class UsersController {
     return this.usersService.findAll();
   }
 
+  @Get('me')
+  async getMe(@Req() req: any): Promise<User> {
+    return this.usersService.findOne(req.user.id);
+  }
+
+  @Get('logout')
+  async logout(@Res() res: any): Promise<void> {
+    res.clearCookie('AuthToken', { secure: true, httpOnly: true });
+    res.redirect('/');
```

Comment:
**question:** does this need to be a redirect? Should be possible to delete the cookies as part of an empty standard response as this is invoked as an API call